src/AcceptanceTesting/AcceptanceTesting.csproj

```diff
@@ -11,7 +11,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="NServiceBus.AcceptanceTesting" Version="9.1.1" />
+    <PackageReference Include="NServiceBus.AcceptanceTesting" Version="9.1.0" />
```

Comment:
Is there a reason for rolling back this version?

Reply:
nope - merge issue - fixed